### PR TITLE
A revised Spacelaw

### PR DESCRIPTION
**Pull request by BRINGit34**
## About the PR
I have written, with the help of many others in the community, a new revised space law. It combines much of the new abbreviated spacelaw with the spacelaw of old. 

Credits for all the help from the community: 
Imperator Nibblez
Paradise Gamerz
Mikki
Clone0401

## Why / Balance
As a frequent player of space-station 14 I can expect to have changes I may not like or disagree with, and that's okay. This is a game with an extremely diverse community and that is why I love it so. 
I am doing this as I believe this will improve many players experience on wizden servers. 

## Technical details
To get into the actual technicalities of what has changed I will create a list that highlights major changes. And a list after that explaining why.

1. I have added a clear definition of what a warrant is and who is authorized to approve one. 
2. I have re-added “Docking and Visitors” in its entirety from SOP, and updated it to accommodate syndicate refugees. 
3. I have re-added the “High Security Areas” list from SOP and removed any mention of when these areas can be bolted.
4. The initial paragraph of sentencing has been rewritten. This version limits an officer's sentencing ability to only sentences under 15 minutes unless HOS & Warden are absent.
5. I have adjusted the permanent brig to be named communal brig
6. I have re-added a listing for "extended confinement" as it was confusing having it missing. 

### Please note this is not the full list of changes, I cleared up some loopholes and changed wording around. 

1. I believe warrants are a great thing to have in space-law, they allow a bit of inter-person discussion to get some paperwork done and are just a fun thing to issue out. My issue with warrants as they are currently is how unspecified they are. 
What do I mean? Warrants as is don't really have a definition. What I have done with my version of warrants is explain what a warrant is. And who can issue them out. This allows for a more consistent warrant system that is not really up for interpretation. 

2. I have mainly added back "docking and visitors" as it is sorely missed now that visitors and docking are relatively common things now and I believe it is best for spacelaw to reflect that. 

3. Leaving what is a high security area up for interpretation, while easier than updating a list has a majority of drawbacks. Namely "what is a high security area?". Is Grav a secure area? Or the research server room? What about AI? These are all left in the air with the current version.
Within my space law holds a list featuring all areas that are considered "secure" and who should have access to them.
I understand that some may like the vagueness of current spacelaw but in all honesty it simply makes spacelaw confusing and hard to comprehend which should be avoided in a spacelaw that will span both LRP and MRP. 

4. Oh boy. This initial paragraph of sentencing. It had stumped me multiple times, it was written in a very confusing way and I still see disagreements over what it actually means. This has gone through a few rewrites on my end and I believe what I have come to now is a far more agreeable version.

Here is my version

> All sentences above 15 minutes must be handled by the Warden or Head of Security. Officers may only sentence above 15 minutes if no Warden or Head of Security is available. As long as those requirements are met, security officers giving inappropriate sentence lengths is considered an in-character issue, not a rule issue. 

5. This was done to be more consistent, and because it makes more sense as a name. 

6. To be honest extended confinement is still in current spacelaw but this makes it far easier to find and understand. 

You may have noticed a lot of these things that were added to make things easier to understand or to clear some things up. I think that in shortening and removing a lot of old space law we have made our new "abbreviated spacelaw" much more confusing. 

credits: 
## Media

Here are images of my major changes. This is also to prove I tested this for any errors

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

Should be zero breaking changes
**Changelog**

:cl: -tweak: Spacelaw has been revised once more

### CODE REVIEW

**Comment by BRINGit34** [comment]
I forgot to mention this but you can see my wiki version as well [here](https://wiki.spacestation14.com/wiki/User:BRINGit34/Space_Law)

**Comment by Nairodian** [comment]
Small request from me because I was going to do it myself anyhow. Could you readd the expediated execution clause as well? Seen: [here](https://wiki.spacestation14.com/w/index.php?title=Space_Law&diff=prev&oldid=5547).

**Comment by Nairodian** [comment]
> So my main issue with this is that execution no longer requires trials. So any need for expedited execution is kind of null.
> 
> I am open to changes though. But it feels to me that captains approval is a totally reasonable thing to need for any execution. Even if it is for refusal of mindshielding.
> 
> And I really do think bringing expedited executions back would bring back reason to have trials for regular executions. And trials are a fairly disliked thing and are only fun for lawyers and those that enjoy that level of rp.

Gah- yeah okay I had a whole thing written out about how in place of not requiring a trial, not requiring approval from the Captain would be better suited, but I apparently accidently deleted it with something else.

As for the whys, Revos is a extremely chaotic gamemode and the chance of you actually having a living Captain who has the time on their hands to give you an answer to your execution question is too rare. Not to mention that _slow is death_ for security and command, as the longer they take to terminate the root threat, the more people turn, and if not that, the already turned are given more time to cause further damage which generally decreases their odds for survival. In short, speed to terminate the root threat in revos is key, as such, expedited executions for shield implants that fail.

**Comment by BRINGit34** [comment]
> > So my main issue with this is that execution no longer requires trials. So any need for expedited execution is kind of null.
> > I am open to changes though. But it feels to me that captains approval is a totally reasonable thing to need for any execution. Even if it is for refusal of mindshielding.
> > And I really do think bringing expedited executions back would bring back reason to have trials for regular executions. And trials are a fairly disliked thing and are only fun for lawyers and those that enjoy that level of rp.
> 
> Gah- yeah okay I had a whole thing written out about how in place of not requiring a trial, not requiring approval from the Captain would be better suited, but I apparently accidently deleted it with something else.
> 
> As for the whys, Revos is a extremely chaotic gamemode and the chance of you actually having a living Captain who has the time on their hands to give you an answer to your execution question is too rare. Not to mention that _slow is death_ for security and command, as the longer they take to terminate the root threat, the more people turn, and if not that, the already turned are given more time to cause further damage which generally decreases their odds for survival. In short, speed to terminate the root threat in revos is key, as such, expedited executions for shield implants that fail.

With revs being such a rare gamemode right now and it in dire need of a rework I don't think I'm willing to implement a sentence that only affects the one crime due to the fact it's possible the change may become useless in time. 

It's the same issue with mass implantation, there is currently no clause for it and I don't see any need for it with revs being so rare. 

If enough people want it I will look at adding it back to this revision.

**Comment by JIPDawg** [comment]
Let's see Chief_Engineer remove himself from a PR this time lol.

**Comment by Emisse** [comment]
Seems fine but needs headmin approval

**Comment by Muolai** [comment]
> s see C

Make it so a sentence over 15-20 min (but not perma) requires a trial or stamps from cap and hos + department head maybe, if a long sentence on first offense is truly needed, then the seccie would need to invest effort, not just stack dubious charges and call it a day. 

Another thing, would it be possible to add a paragraph on confiscation of non explicit contraband items in connection with a crime. ie a tider can get gets his insuls confiscated for trying to hack a door. (this is already precedent from admin questions)

**Comment by BRINGit34** [comment]
> Make it so a sentence over 15-20 min (but not perma) requires a trial or stamps from cap and hos + department head maybe, if a long sentence on first offense is truly needed, then the seccie would need to invest effort, not just stack dubious charges and call it a day.

Well most sentencing will be done by HOS or Warden. Not officers. The first  paragraph of sentencing states as much. Any sentence over 15 minutes will be extended confinement and should only be possible with stacked crimes. This extended confinement will keep the  prisoner in the communal brig instead of their own cell. 

And I am trying to avoid trials, they take longer than sentences most of the time and most people don't enjoy trial RP. And cap's approval for a long sentence seems extreme. Caps approval should only be needed for execution.

> Another thing, would it be possible to add a paragraph on confiscation of non explicit contraband items in connection with a crime. ie a tider can get gets his insuls confiscated for trying to hack a door. (this is already precedent from admin questions)

Could you link me that and I'll try to add it in here somewhere

**Comment by Muolai** [comment]
> Could you link me that and I'll try to add it in here somewhere

image

**Comment by BRINGit34** [comment]
> > Could you link me that and I'll try to add it in here somewhere
> 
> image

I'll add a clause about that later today

**Comment by Muolai** [comment]
would it also be possible to add a law clarifying how sec equipment is supposed to be distributed in case of emergency- ie command can only carry lasers/ sechunds on red and nonlethals on blue to minimize powergaming.

**Comment by Golinth** [comment]
> A syndie rolls a jug suit + bulldog from a surplus crate and gets stuned by a robust det in maints => Possession/Use of Syndicate Contraband (10 min)

Just to be nitpicky, the bulldog would extend this sentence by 5 minutes, upgrading it from possession of restricted gear to possession of restricted weaponry, for a 15 minute maximum.

**Comment by Muolai** [comment]
> > A syndie rolls a jug suit + bulldog from a surplus crate and gets stuned by a robust det in maints => Possession/Use of Syndicate Contraband (10 min)
> 
> Just to be nitpicky, the bulldog would extend this sentence by 5 minutes, upgrading it from possession of restricted gear to possession of restricted weaponry, for a 15 minute maximum.

highest tier for Possession is 3 so the max sentence is always 10 min, no clue where you got that from....

**Comment by Golinth** [comment]
> highest tier for Possession is 3 so the max sentence is always 10 min, no clue where you got that from....

😬 ignore me, I was confused by the naming there. Though you have the lower time, not the higher. Whoopsy

**Comment by BRINGit34** [comment]
> Most sentencing is done by an officer stating some crimes and the HOS giving it a blind yes, because the hos is not on the side of the prisoner. We already have enough shitters /ghosting after they are put into perma or a 20 minute sentance as the prison is very boring, the goal with my suggestion is to make an extended sentace have to have merit/oversight instead of being an exercise of stacking crimes. here are some examples so you can decide who deserves a longer sentance:

A lot of this seems more like a community issue than that of a spacelaw issue. It is up to the HOS or  most of the time Warden to decide how to sentence and for lawyers to argue against that sentence. And it would be doable to somewhat lower the maximum time for extended confinement. 45 was what it used to be at. 30 may be a more believable number for MRP & LRP. 
That would be just 15 minutes longer than a full brig sentence which seems quite doable. Especially since it will be spent in the communal brig. 


>     * Shiter walks with the cap in a command room and tries to gun him down (he cracks a window after missing a bunch) with a viper and walks out of the cell for 2 seconds because security was incompetent with closing the door -> Murder (15 min) + Breach of Custody (15 min) + Secure Trespass (10 min) + Possession/Use of Syndicate Contraband(10 min) + Vandalism (5 min) = 55 min
> 
>     * A thief medic follows the cmo into his room and distracts him to grab his hipospray, after a chase he is detained -> Failure to Comply (5 min) + Possession/Use of Major Contraband (5 min) + Secure Trespass (10 min) + Grand Theft(10 min) = 30 min
> 
>     * A syndie rolls a jug suit + bulldog from a surplus crate and gets stuned by a robust det in maints => Possession/Use of Syndicate Contraband (10 min)
> 
>     * A thief is found with the id of every command member on the station -> Grand Theft (10 min)
> 
>     * A robust shitter does not comply with a search warrant, slips a bunch of seccies and causes a station wide manhunt, is breaking out of the cell, brawling in sec multiple times, and being an absolute asshole, but not enough to perma ->Breach of Custody (15 min)

For the last one specifically I want to point out that you would probably be able to perma someone for multiple breach of custody charges. If I am understanding this what-if correctly. 

But a lot of this is just an example of someone doing more crime and doing more time. Which seems like a reasonable expectation. 

If a player chooses to enable playing as a syndicate operative they should be willing to spend time in a cell. 

> 
> If trials are not wanted at least make it so you **have** to ask the qm if they are fine with having half their department get put into the hole for 45 min for a first offence, even if they are all syndies. (this would pervent a whole department from getting extended senteces and for someone who is neutral/ on the side of the prisoner to have a say in the sentencing).

It's not the QM's job to be a lawyer, nor should it be. This also seems like an unlikely scenario to have happened. Also not every command member knows space law or wants to interact with space law at all. 

This just seems like a lot of loops for a spacelaw that will operate on both LRP & MRP

**Comment by Muolai** [comment]
> A lot of this seems more like a community issue than that of a spacelaw issue.

Yeah, it is totally a community issue, we cannot change the community but we can amend the space law to get a more favorable rp environment.

> If a player chooses to enable playing as a syndicate operative they should be willing to spend time in a cell.

Would you want to spend 45 minutes of your life in a space rp game sitting in a grey box growing pixel vegetables? (this is also a critique of the wizden perma cells)
In MRP this should be fine but for LRP i think 45 min is a bit much.

> But a lot of this is just an example of someone doing more crime and doing more time. Which seems like a reasonable expectation.
i was trying to demonstrate how sentencing becomes a game of seeing how many check boxes you can tag instead of an assessment of danger or future risk to the station or sec.


> It's not the QM's job to be a lawyer, nor should it be. This also seems like an unlikely scenario to have happened. Also not every command member knows space law or wants to interact with space law at all.
> 
> This just seems like a lot of loops for a spacelaw that will operate on both LRP & MRP

Fair point.

**Comment by BRINGit34** [comment]
> Would you want to spend 45 minutes of your life in a space rp game sitting in a grey box growing pixel vegetables? (this is also a critique of the wizden perma cells)
In MRP this should be fine but for LRP i think 45 min is a bit much.

I am amending this to be 30. The 45 minutes was honestly an oversight on my part. 

>i was trying to demonstrate how sentencing becomes a game of seeing how many check boxes you can tag instead of an assessment of danger or future risk to the station or sec.

I feel that for sentencing to be anything but tagging check boxes it will need a serious re-work and that is not my intention with this. I just want to amend our current space law to be a more enjoyable and understandable version of itself.

**Comment by BRINGit34** [comment]
I have implemented formatting changes to make things easier to read. 

better


As well as implementing changes recommended through discussion such as lowering the maximum extended confinement time and implementing confiscating items used in a crime.

**Comment by BRINGit34** [comment]
I also have noticed I made an error in my initial "technical details" on item 4. 

This has now been fixed. Officers can sentence as long as the sentence does not exceed 15 minutes. I mixed up this version with an older version I was working on. 

Sorry about that

**Comment by BRINGit34** [comment]
> Seems fine but needs headmin approval

What would be the best way to go about getting this?

**Comment by Muolai** [comment]
> > Seems fine but needs headmin approval
> 
> What would be the best way to go about getting this?

have you asked for a pr-review in the #pr-review  discord channel?

**Comment by BRINGit34** [comment]
> > > Seems fine but needs headmin approval
> > 
> > 
> > What would be the best way to go about getting this?
> 
> have you asked for a pr-review in the #pr-review discord channel?

Yeah a few days ago. But that's alright. I'll just wait for some more time

File: Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SLCrimeList.xml [hunk 1]
```diff
@@ -716,7 +716,7 @@
 
   ## Extreme Crimes
   - The maximum sentence per extreme crime is 15 minutes.
-  - Particularly violent offenders may be placed in perma. (Attacking officers in an attempt to escape prison)
+  - Particularly violent offenders may be placed in permanent confinment. (Attacking officers in an attempt to escape prison)
```
Comment (slarticodefast) [inline Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SLCrimeList.xml:719]:
```suggestion
  - Particularly violent offenders may be placed in permanent confinement. (Attacking officers in an attempt to escape prison)
```
typo

**Comment by LordEclipse** [comment]
> > Small request from me because I was going to do it myself anyhow. Could you readd the expediated execution clause as well? Seen: [here](https://wiki.spacestation14.com/w/index.php?title=Space_Law&diff=prev&oldid=5547).
> 
> So my main issue with this is that execution no longer requires trials. So any need for expedited execution is kind of null.
> 
> I am open to changes though. But it feels to me that captains approval is a totally reasonable thing to need for any execution. Even if it is for refusal of mindshielding.
> 
> And I really do think bringing expedited executions back would bring back reason to have trials for regular executions. And trials are a fairly disliked thing and are only fun for lawyers and those that enjoy that level of rp.

Well expedited executions were added for a reason. Revs is very difficult to contain, and it would apply outside of just revs (cults, any other mindcontrol based conversion antag) so it is also future proofing. It allows players to act upon an obvious decision right in front of them without the need for tactless bureacracy. A Captain is not always available and needing to require approval for somebody that everybody at that point KNOWS if a head rev/head cultist is just adding unnecessary legwork. It weakens Space Law for something with an obvious outcome.

**Comment by Titian3** [comment]
You have walked in to hell with suggesting rule changes sir. Good luck.
If it does get towards making a change. You will probably have to make a update for the [wiki](https://wiki.spacestation14.com/wiki/Space_Law) too.

> I have re-added the “High Security Areas” list from SOP and removed any mention of when these areas can be bolted.
Couldn't see where this was removed in the diff but it sounds like power gaming to me.

**Comment by BRINGit34** [comment]
> > > Small request from me because I was going to do it myself anyhow. Could you readd the expediated execution clause as well? Seen: [here](https://wiki.spacestation14.com/w/index.php?title=Space_Law&diff=prev&oldid=5547).
> > 
> > 
> > So my main issue with this is that execution no longer requires trials. So any need for expedited execution is kind of null.
> > I am open to changes though. But it feels to me that captains approval is a totally reasonable thing to need for any execution. Even if it is for refusal of mindshielding.
> > And I really do think bringing expedited executions back would bring back reason to have trials for regular executions. And trials are a fairly disliked thing and are only fun for lawyers and those that enjoy that level of rp.
> 
> Well expedited executions were added for a reason. Revs is very difficult to contain, and it would apply outside of just revs (cults, any other mindcontrol based conversion antag) so it is also future proofing. It allows players to act upon an obvious decision right in front of them without the need for tactless bureacracy. A Captain is not always available and needing to require approval for somebody that everybody at that point KNOWS if a head rev/head cultist is just adding unnecessary legwork. It weakens Space Law for something with an obvious outcome.

Alright I'll reincorporate it back into space law. 

I should have that done later today

**Comment by Titian3** [comment]
> > I can kinda see an issue with a shitsec always doing max sentencing and that could become an admin issue as it's pretty shit, but overall I think is fine
> 
> Make it so a sentence over 15-20 min (but not perma) requires a trial or stamps from cap and hos + department head maybe, if a long sentence on first offense is truly needed, then the seccie would need to invest effort, not just stack dubious charges and call it a day.
> 
> Another thing, would it be possible to add a paragraph on confiscation of non explicit contraband items in connection with a crime. ie a tider can get gets his insuls confiscated for trying to hack a door. (this is already precedent from admin questions)

This, TBH. No difference between an execution and perma/long sentences in most cases currently as they both end up with a dead mob in the cells and another ghost.

**Comment by BRINGit34** [comment]
> You have walked in to hell with suggesting rule changes sir. Good luck. If it does get towards making a change. You will probably have to make a update for the [wiki](https://wiki.spacestation14.com/wiki/Space_Law) too.

I hope it goes well. And there is already a wiki page for this version. [It should be up to date](https://wiki.spacestation14.com/wiki/User:BRINGit34/Space_Law#Vessels)

> > I have re-added the “High Security Areas” list from SOP and removed any mention of when these areas can be bolted.
> 
> Couldn't see where this was removed in the diff but it just sounds like power gaming to me if you are bolting up all the secure doors for no reason, they already have game mechanics to make them harder to break in to.

Well in the sop there used to be restrictions on when something could be bolted. But that still seems like an SOP issue instead of a space law issue.

**Comment by BRINGit34** [comment]
> > > I can kinda see an issue with a shitsec always doing max sentencing and that could become an admin issue as it's pretty shit, but overall I think is fine
> > 
> > 
> > Make it so a sentence over 15-20 min (but not perma) requires a trial or stamps from cap and hos + department head maybe, if a long sentence on first offense is truly needed, then the seccie would need to invest effort, not just stack dubious charges and call it a day.
> > Another thing, would it be possible to add a paragraph on confiscation of non explicit contraband items in connection with a crime. ie a tider can get gets his insuls confiscated for trying to hack a door. (this is already precedent from admin questions)
> 
> This, TBH. No difference between an execution and perma/long sentences in most cases currently as they both end up with a dead mob in the cells and another ghost.

I reduced the maximum length of extended confinement to 30. I hope that this will reduce some of the ghosting that is prevalent in long sentences.

**Comment by BRINGit34** [comment]
# Expedited Executions 
--- 
I have re-implemented them with some rephrasing to accommodate the new phrasing of executions, also so it makes some more sense.

Screenshot from 2024-08-28 17-45-29

And I have made sure to mention it in the notes of Refusal of Mind Shielding

Screenshot from 2024-08-28 17-47-54

**Comment by BRINGit34** [comment]
# Just to add to this some

The server Harmony uses part of this space law in their ["Warrants" section](http://wiki.harmony14.com/index.php/Space_Law#Warrants)

So if there are any concerns to how well that section would work in-game then it may be good to ask their team and see what they think.

**Comment by BRINGit34** [comment]
Screenshot_20250118-081125
Screenshot_20250118-081136

After reading this I am considering adding some sort of clause for this. 

Rev-rework seems a while off and this seems like a good compromise

File: Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml [hunk 2]
```diff
@@ -11,49 +11,89 @@
...
-
-  A departmental search is the sweep of an entire area or department for contraband. It is recommended that the officers be extremely thorough, checking all lockers, crates, and doors. These can only be done with permission or, ideally, a warrant signed by the department head or highest-ranking command staff, which is the captain in most cases.
+<Box>[color=#a4885c]Personnel search[/color]</Box>
+  A personnel search is a seizure of the objects in a person's backpack, hands, coat, belt, and pockets. 
+  
+  - If any contraband is found during a search, the officer may choose to further the search into a detainment or simply confiscate the restricted items. 
+  - If an item that was used in a crime is found during a search the officer may choose to confiscate it, with a reasonable amount of evidence. 
+
+  After the search is conducted, all legal items are to be returned to the person. A crewmate may legally decline any search conducted without probable cause or a warrant while the alert level is green. It should be noted that if the alert level is blue or above, all personnel searches are legal.
```
Comment (Hrosts) [inline Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml:25]: The wording is confusing as whether a forceful search requires _both_ probable cause and a warrant, or _either_ a probable cause or a warrant.

Comment (BRINGit34) [reply Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml:25]: Which part of the wording exactly is confusing? [Or](https://www.merriam-webster.com/dictionary/or) used as a conjunction indicates an alternative. So either a warrant or probable cause is needed for a search under green alert. 

If both were required it would be written "A crewmate may legally decline any search conducted without probable cause _and_ a warrant" 

But maybe I am missing the part that is confusing.

Comment (Hrosts) [reply Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml:25]: > A crewmate may legally decline any search conducted without probable cause or a warrant
>So either a warrant or probable cause is needed for a search under green alert
Or either a warrant or probable cause need to be missing for me to safely reject the search.

Comment (BRINGit34) [reply Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml:25]: But it doesn't state both are needed anywhere. Besides this is how it's been written for a very long time and I personally have very rarely seen confusion about what constitutes a search under green alert, except in cases of someone just learning space law for the first time.

Comment (Hrosts) [reply Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml:25]: I don't think I have ever seen a code green search through a probable cause. It's either a warrant, or they switch to blue and search regardless.
I have also seen people demand a warrant where there is a probable cause and have sec back off. I'm not sure many people know about the probable cause clause.

It's not that hard to rephrase it to explicitly state "either".

Comment (BRINGit34) [reply Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml:25]: Any time someone is arrested they are searched through probable cause. And if sec backs off even though they have probable cause that is an IC issue. Nothing to do with the wording of space law. I see it in use every shift as security, someone is arrested, they are searched without a warrant but with probable cause. 

I don't see reason to change wording that has been used for a long time and is fairly agreed upon. 

Here is a case of admins using seeing a crime be committed as an example of probable cause 
Screenshot From 2025-01-19 10-06-53

Comment (Hrosts) [reply Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml:25]: Whatever, I guess. Maybe I'm wrong. This bit definitely looked like something you could lawyer your way into demanding a warrant every time.
image

**Comment by BramvanZijp** [comment]
Might help to allow sec to perma people if the sentence would exceed 30 minutes, considering you replaced the 15 minute barrier with extended confinement.

I also still believe spacelaw as a whole remains flawed, so I am working on a full rewrite.

**Comment by BRINGit34** [comment]
I'm closing this pr. I don't really play the game much right now and I don't really want to work on this further. 

If anyone wants to use this pr for something, go ahead by all means.
